core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -555,19 +561,21 @@ class KafkaController(val config: KafkaConfig,
    * A reassignment may have up to three phases, each with its own steps:
    *
    *
-   * Cleanup Phase: In the cases where this reassignment has to override an ongoing reassignment.
+   * Cleanup Phase: In the cases where this reassignment has to override or cancel an ongoing reassignment.
    *   . The ongoing reassignment is in phase A
    *   . ORS denotes the original replica set, prior to the ongoing reassignment
-   *   . URS denotes the unnecessary replicas, ones which are currently part of the AR of the ongoing reassignment but will not be part of the new one
-   *   . OVRS denotes the overlapping replica set - replicas which are part of the AR of the ongoing reassignment and will be part of the overriding reassignment
-   *       (it is essentially (RS - ORS) - URS)
+   *   . URS denotes the unnecessary replicas, ones which are currently part of the AR of the ongoing
+   *     reassignment but will not be part of the new one
+   *   . OVRS denotes the overlapping replica set - replicas which are part of the AR of the ongoing
+   *     reassignment and will be part of the overriding reassignment (it is essentially (RS - ORS) - URS)
    *
-   *   1 Set RS = ORS + OVRS, AR = OVRS, RR = [] in memory
-   *   2 Send LeaderAndIsr request with RS = ORS + OVRS, AR = OVRS, RR = [] to all brokers in ORS + OVRS
+   *   1 Set RS = ORS + OVRS, AR = OVRS, RR = [] in zk
+   *   2 Set RS = ORS + OVRS, AR = OVRS, RR = [] in memory
+   *   3 Send LeaderAndIsr request with RS = ORS + OVRS, AR = OVRS, RR = [] to all brokers in ORS + OVRS
```

Comment:
The cleanup phase referred to the `maybeRevertOngoingReassignment` method. Now that we don't have it, I think we need to remove this section